### ISSUE TITLE
doesnt work

### ISSUE THREAD

**Issue by fakhry7050** [H1B0B0/Kick-Viewerbot#6]
i download exe and i put list proxy when click button start bot doesnt happen

**Comment by H1B0B0** [comment]
Yes, the difference between the Twitch Viewer bot and the Kick Viewer bot is significant. I need to perform reverse engineering to understand how the human verification works on Kick. You can try running the code now; it's not very difficult. I can update the .bat files.

**Comment by eliecer696** [comment]
the kick api you were using from yt-dlp seems deprecated

**Comment by H1B0B0** [comment]
A new version is here :)
